packages/expo/src/Expo.ts

```diff
@@ -17,7 +16,6 @@ export { default as Notifications } from './Notifications/Notifications';
 
 // The following should all be extracted from this package
 export { AR };
-export { ScreenOrientation };
```

Comment:
We need to warn our users about this change - please see on the big export below and `removed.ts` file and put `ScreenOrientation` there